apps/web/playwright/reschedule.e2e.ts

```diff
@@ -148,7 +145,6 @@ test.describe("Reschedule Tests", async () => {
     });
 
     await expect(page).toHaveURL(/.*payment/);
-    await payment.delete();
```

Comment:
Since the old booking is deleted, the old payment is also deleted. A new payment record is created with the new booking